tcf_website/migrations/0024_forumcategory_forumpost_forumresponse_question_title_and_more.py

```
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.28 on 2026-02-04 05:13
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("tcf_website", "0023_remove_sectionenrollment_section_and_more"),
+    ]
```

Ruff (0.15.11)

[warning] Mutable default value for class attribute

(RUF012)